From 670090027d6cb61f95ef88fc35bc309117a04a94 Mon Sep 17 00:00:00 2001
From: jchinlee <jchinlee94@gmail.com>
Date: Mon, 10 Jul 2023 15:39:02 -0700
Subject: [PATCH 07/10] Update COPYING with copyright line

The MIT license requires a copyright line specifying the year of application and copyright holder. [0]

Add this line to the existing COPYING file containing the rest of the license text.

[0] https://opensource.org/license/mit/
---
 COPYING | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/COPYING b/COPYING
index 900e775..6270fa9 100644
--- a/COPYING
+++ b/COPYING
@@ -1,3 +1,5 @@
+    Copyright (c) 2023 Intel Corporation
+
     Permission is hereby granted, free of charge, to any person obtaining a
     copy of this software and associated documentation files (the
     "Software"), to deal in the Software without restriction, including
-- 
2.47.1.windows.1

